### ISSUE TITLE
Go style suggestion

### ISSUE THREAD

**Issue by Vorror** [bchiang7/halcyon-vscode#18]
Currently, when using one of the supported language(e.g Javascript), there's a nice mix of colors. However, when using Go, the color blue is noticeably absent. I recommend changing the color of ```error``` value blue:

![image](https://user-images.githubusercontent.com/3613230/63230623-58027500-c1dd-11e9-8f1c-caa1b40b40e0.png)

Code:
```js
{
          "scope": [
            "storage.type.error.go"
          ],
          "settings": {
            "foreground": "#5ccfe6",
          }
}
```

For reference, here's what Go looks like without the change:
![before](https://user-images.githubusercontent.com/3613230/63230668-e545c980-c1dd-11e9-8ec3-4ea4ba23da79.png)

**Comment by Vorror** [comment]
The reference code above was taken from [Plastic's Theme repo](https://github.com/will-stone/plastic/blob/master/samples/go.go)

**Comment by Vorror** [comment]
Another suggestion is to add:

```js
 {
          "scope": "punctuation.other.period.go",
          "settings": {
            "foreground": "#ffae57",
          }
        },
```
So Go gets the period color like other languages.

![image](https://user-images.githubusercontent.com/3613230/63230712-816fd080-c1de-11e9-9d23-b91d67b3943f.png)

**Comment by bchiang7** [comment]
Thanks for the detailed suggestions! I don't write any go so I'm open to any feedback you may have. Thanks for using this theme!

Addressed in #19 and `v0.2.6`
